packages/layout/src/components/cards/common/views/preview/components/companiesHouseNumber/companiesHouseNumber.tsx

```diff
@@ -12,7 +12,7 @@ export const CompaniesHouseNumber: React.FC<ICompaniesHouseNumberProps> = React.
 		return (
```

Comment:
`houseNumber` is a bit of a confusing property name. You'll find a house number outside your front door, but Companies House is the name of an organisation. It's never known as just "house".